modules/parquet/test/data/files.js

```diff
@@ -6,23 +6,23 @@ const PARQUET_FILES = [
   {supported: true, title: 'alltypes_plain', path: 'good/alltypes_plain.parquet'},
   {supported: true, title: 'alltypes_plain_snappy', path: 'good/alltypes_plain.snappy.parquet'},
   {supported: true, title: 'binary', path: 'good/binary.parquet'},
-  {supported: false, title: 'bloom_filter', path: 'good/bloom_filter.bin'},
+  {supported: false, title: 'bloom_filter', path: 'good/bloom_filter.bin'}, // It's not a parquet file should we remove it?
   {supported: false, title: 'byte_array_decimal', path: 'good/byte_array_decimal.parquet'},
   {supported: false, title: 'datapage_v2', path: 'good/datapage_v2.snappy.parquet'}, // Doesn't work on parquet-tools
   {supported: true, title: 'dict', path: 'good/dict-page-offset-zero.parquet'},
   {supported: false, title: 'fixed_length_decimal', path: 'good/fixed_length_decimal.parquet'},
   {supported: false, title: 'fixed_length_decimal_legacy', path: 'good/fixed_length_decimal_legacy.parquet'},
-  {supported: false, title: 'hadoop_lz4_compressed', path: 'good/hadoop_lz4_compressed.parquet'},
-  {supported: false, title: 'hadoop_lz4_compressed_larger', path: 'good/hadoop_lz4_compressed_larger.parquet'},
+  {supported: false, title: 'hadoop_lz4_compressed', path: 'good/hadoop_lz4_compressed.parquet'}, // Doesn't work on parquet-tools
```

Comment:
LZ4 seems to be fairly new addition to parquet. We do have an LZ4 decompressor, but parquet seems to use LZ4 compression without header so we need to find a way to e.g. add a dummy header.

Reply:
Cool, Thanks for the info. Will do in separate PR.

Reply:
Left a comment there.